service/src/LevelZero.cpp

```diff
@@ -601,6 +602,30 @@ namespace geopm
         return {property.min, property.max};
     }
 
+    std::vector<double> LevelZeroImp::frequency_supported(unsigned int l0_device_idx,
```

Comment:
Maybe not a huge priority.  I understand why the LevelZeroDevicePool would return frequency in MHz if that is the units provided by the underlying implementation.  Why, however, doesn't the LevelZero class use SI units?